docs/deployment.rst

```diff
@@ -1,6 +1,168 @@
 E4S Deployments
 ===============
 
+Crusher
+----------
+
+For more information on Crusher please see https://docs.olcf.ornl.gov/systems/crusher_quick_start_guide.html
+
+**ParaTools Deployment: E4S 22.05 / PrgEnv-gnu**
+
+This deployment is maintained by `ParaTools <https://www.paratools.com/>`_ and may not be officially supported by facility staff.
+
+This deployment is built using Spack at branch `e4s-22.05` and the following facility-provided external packages:

```

Comment:
so the spack project is named `spack` and not `Spack` with a capital "S", i think it would be better if we can be consistent and use `spack` throughout the documentation and use capital "S" when its at start of word.

Reply:
Spack is the name of the tool. See https://github.com/spack/spack at the very top of README. It is a name, so it would be capitalized.

Reply:
yeah i think they just capitalized it in the README. We had run into similar issue when writing our user-facing docs for docs.nersc.gov and it become quite confusing when and where to u.

It looks like even spack doesn't follow the same convention though i do see most instances of `Spack` that start sentence are capitalize 

![Screen Shot 2022-07-28 at 5 38 11 PM](https://user-images.githubusercontent.com/12942230/181642064-f309734a-c13a-44aa-abeb-fa376fd449b7.png)

I did a search on https://spack.readthedocs.io/en/latest/getting_started.html and it looks like most instance are capitalize of `Spack` even in middle of sentence so i guess we should keep it capitalize. I find this quite confusing tbh.

Reply:
im going to resolve this issue since `Spack` is used in most of their docs as pose to `spack`